frameworks/elastic/src/main/dist/svc.yml

```diff
@@ -49,17 +49,7 @@ pods:
           path: "container-path"
           type: {{MASTER_NODE_DISK_TYPE}}
           size: {{MASTER_NODE_DISK}}
-        cmd: |
-          set -e
-
-          export JAVA_HOME=$(ls -d $MESOS_SANDBOX/jdk*/jre/)
-          ./elastic-scheduler/java-dns-ttl.sh
-          ./elastic-scheduler/install-plugins.sh
-          ./elastic-scheduler/setup-keystore-bootstrap-password.sh
-          ./elastic-scheduler/setup-java-policy.sh
-          ./bootstrap
-
-          exec ./elasticsearch-{{ELASTIC_VERSION}}/bin/elasticsearch
+        cmd: "export JAVA_HOME=$(ls -d $MESOS_SANDBOX/jdk*/jre/) && ./elastic-scheduler/java-dns-ttl.sh && ./elastic-scheduler/install-plugins.sh && ./elastic-scheduler/setup-java-policy.sh &&  ./bootstrap && exec ./elasticsearch-{{ELASTIC_VERSION}}/bin/elasticsearch"
```

Comment:
Do we want to do a clean revert, or could we leave the "improvements" such as this?

Reply:
@elezar I pushed a commit that makes this PR keep all non-actual-upgrade improvements: https://github.com/mesosphere/dcos-commons/pull/2274/commits/31a2cb45b47861c7c55d5013485378bc951b0551